behavior.nls

```diff
@@ -85,6 +85,74 @@ to update-landscape
   ]
 end
 
+to update-memories-only 	; procedure to be performed before every movement by the respective agent so that she always remembers the arguments she saw
```

Comment:
We should find a common commenting style:
my suggestion is that procedures are preceeded by comments
describing what the respective procedure does.
So better put your comment above the "to update-memories-only"